.changeset/angry-windows-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ithaka/pharos-site': major
```

Comment:
This seems like a patch rather than a major change?

Reply:
Thanks for pointing it out, @daneah . Fixed it.